packages/strapi-plugin-content-manager/admin/src/containers/ListView/index.js

```diff
@@ -118,7 +118,7 @@ function ListView({
   const [isFilterPickerOpen, setFilterPickerState] = useState(false);
   const [idToDelete, setIdToDelete] = useState(null);
   const contentType = layout.contentType;
-  const hasDraftAndPublish = contentType.options.draftAndPublish;
+  const hasDraftAndPublish = contentType.options.draftAndPublish || false;
```

Comment:
The get function of lodash is already imported in this file. Can you use it here, please?
 
`const hasDraftAndPublish = get(contentType, ['options', 'draftAndPublish'], false);`